### PR TITLE
Add mobile menu functionality

### PR DESCRIPTION
**Pull request by hansderly**
In this PR, we:
- Added linter for Javascript files
- Add menu for mobile version
- Added closing functionality when the user clicks (or taps) the hamburger button
- Added closing functionality when the user clicks (or taps) the close (X) button.
- Added closing functionality when the user clicks (or taps) any of the mobile menu options.
- Added closing functionality when the user clicks (or taps) any of the mobile menu options.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,63 +27,78 @@
...
+                                    fill="#93A79D" />
+                            </svg>
+                        </div>
+                        <ul id="mobile-links">
+                            <li><a href="#portfolio">Portfolio</a></li>
+                            <li><a href="#about">About</a></li>
+                            <li><a href="#contact-form">Contact</a></li>
+                        </ul>
+                    </div>
+                </div>
```
Comment (ShadyShawkat) [inline index.html:49]: - [ ] Please make sure that the mobile menu takes 100% of the screen's height to match the given design.
image

Comment (ShadyShawkat) [inline index.html:49]: - [ ] Kindly make sure that the user cannot scroll in the page itself when the mobile menu is open. Currently, if you have the mobile menu open, you can scroll past it onto the page content.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi Team,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**